yarn.lock

```diff
@@ -10311,6 +10311,11 @@ arch@^2.2.0:
   resolved "https://registry.yarnpkg.com/arch/-/arch-2.2.0.tgz#1bc47818f305764f23ab3306b0bfc086c5a29d11"
   integrity sha512-Of/R0wqp83cgHozfIYLbBMnej79U/SVGOOyuB3VVFv1NRM/PSFMK12x9KVtiYzJqmnU5WR2qp0Z5rHb7sWGnFQ==
 
+arch@^3.0.0:
```

Comment:
clipboardy on the cypress-example-kitchensink requires 2.2.0 still in our yarn lockfile